script.js

```diff
@@ -1,4 +1,435 @@
+  }
+  if (input == "AI") {
+    currentGameMode = "AI";
+    myOutputValue = `You have selected Computer Vs AI as your game of choice! <br><br>An AI machine will pick for you!<br><br>Start by pressing Submit`;
+  }
+  return myOutputValue;
+};
+
+// ------------------ Normal game mode ----------------------
+
+var normalGame = function (input) {
+  // to generate computer's move. console log to see response and verify outcome.
+  var computerMove = randomItem();
+  console.log("computer", computerMove);
+
+  // to generate the icons for the moves
+  var iconComputer = iconLogo(computerMove);
+  console.log("iconComputer", iconComputer);
+  var iconPlayer = iconLogo(input);
+  console.log("iconPlayer", iconPlayer);
+
+  // to call the win, draw, invalid functions in main to help determine outcome.
+  var draw = drawGame(input, computerMove);
+  var win = playerWin(input, computerMove);
+  var tryAgain = invalid(input);
+
+  //tracking of game progress
+  numGamesPlayed += 1;
+  console.log("numGamesPlayed", numGamesPlayed);
+
+  //set base output as lose scenario
+  var myOutputValue = `The computer played ${computerMove} ${iconComputer} .<br>You played ${input} ${iconPlayer}.<br>You lost ='( <br>${userName}, you have played ${numGamesPlayed} games and your win percentage is ${percentageWin()}%.<br>Lets play another round!`;
+
+  // IF condition to determine if player has input an invalid guess with corresponding output. To minus 1 game to counter the default +1 above
+  if (tryAgain) {
+    numGamesPlayed -= 1;
+    myOutputValue = `Please type in scissors, paper or stone.`;
+  }
+
+  // IF condition to determine if player wins with corresponding output.
+  if (win) {
+    numGamesWon += 1;
+    myOutputValue = `The computer played ${computerMove} ${iconComputer}.<br>You played ${input} ${iconPlayer}.<br>You won!!!<br>${userName}, you have played ${numGamesPlayed} games and your win percentage is ${percentageWin()}%.<br>Lets play another round!`;
+  }
+
+  // IF condition to determine if game is a draw with corresponding output.
+  if (draw) {
+    numGamesDraw += 1;
+    myOutputValue = `The computer played ${computerMove} ${iconComputer}.<br>You played ${input} ${iconPlayer}.<br>It's a draw!<br>${userName}, you have played ${numGamesPlayed} games and your win percentage is ${percentageWin()}%.<br>Lets play another round!`;
+  }
+  console.log("numGamesPlayed after ifs", numGamesPlayed);
+  console.log("numGamesWon after", numGamesWon);
+  return myOutputValue;
+};
+
+// -----------------Reversed game mode -------------------
+// -------------------------------------------------------
+
+// notes will only show the difference from Normal game mode
+
+var reversedGame = function (input) {
+  var computerMove = randomItem();
+  console.log("computer", computerMove);
+
+  var iconComputer = iconLogo(computerMove);
+  console.log("iconComputer", iconComputer);
+  var iconPlayer = iconLogo(input);
+  console.log("iconPlayer", iconPlayer);
```

Comment:
Nice. This is great dynamic way to update values even if we add more items other than scissors, paper, stone. Very clean.